etl_entities/hwm_store/hwm_store_manager.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright 2023 MTS (Mobile Telesystems)
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from __future__ import annotations
+
+from typing import ClassVar
+
+from etl_entities.hwm_store.base_hwm_store import BaseHWMStore
+from etl_entities.hwm_store.hwm_store_class_registry import HWMStoreClassRegistry
+
+
+class HWMStoreManager:
```

Comment:
Может заодно стоит переименовать класс?
```suggestion
class HWMStoreStack:
```
или
```suggestion
class HWMStoreStackManager:
```